packages/devtools_app/lib/src/screens/debugger/call_stack.dart

```diff
@@ -89,10 +89,12 @@ class _CallStackState extends State<CallStack>
                   ? theme.subtleTextStyle
                   : theme.regularTextStyle),
           children: [
-            TextSpan(
-              text: ' $locationDescription',
-              style: selected ? theme.selectedTextStyle : theme.subtleTextStyle,
-            ),
+            if (locationDescription != null)
+              TextSpan(
+                text: ' $locationDescription',
+                style:
+                    selected ? theme.selectedTextStyle : theme.subtleTextStyle,
```

Comment:
at master this text style is:

```
              style: selected
                  ? theme.selectedSubtleTextStyle
                  : theme.subtleTextStyle,
```

maybe a merge is needed

Reply:
Done!